src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -0,0 +1,69 @@
+= Using custom domains
+:page-slug: custom-domains
+:page-description: How to configure your Koji to use a custom domain.
+
+You can assign a custom domain name to your Koji after publication.
+A custom domain can have the advantage of being easier to remember, and it usually just looks better.
+Custom domains are about where you want to serve the experience.
+
+Koji has a few subdomains that you can use at no cost and with no additional configuration, including:
```

Comment:
These are the domains that Koji owns. You can use a subdomain on these domains. Subdomains and domains are used inaccurately here and in the first procedure. E.g., step 7 talk about the "name that you want to come before the subdomain." That name _is_ the subdomain, and it comes before the Koji-owned root domain. Clean up the references in both places.